lib/cli/src/commands/create_exe.rs

```diff
@@ -557,35 +548,51 @@ impl CreateExe {
             header_code_path = std::env::current_dir()?;
         }
 
+        println!("Output result to: {}", output_path.display());
+
         /* Compile main function */
         let compilation = {
             let mut include_dir = libwasmer_path.clone();
             include_dir.pop();
             include_dir.push("include");
 
             let mut cmd = Command::new(zig_binary_path);
-            let mut cmd_mut: &mut Command = cmd
-                .arg("cc")
-                .arg("-target")
-                .arg(&zig_triple)
-                .arg(&format!("-L{}", libwasmer_path.display()))
-                .arg(&format!("-l:{}", lib_filename))
-                .arg(&format!("-I{}", include_dir.display()))
-                .arg(&format!("-I{}", header_code_path.display()));
-            if !zig_triple.contains("windows") {
-                cmd_mut = cmd_mut.arg("-lunwind");
+            cmd.arg("build-exe");
+            cmd.arg("-target");
+            cmd.arg(&zig_triple);
+            cmd.arg(&format!("-I{}/", include_dir.display()));
+            cmd.arg(&format!("-I{}/", header_code_path.display()));
+            if zig_triple.contains("windows") {
+                cmd.arg("-lc++");
+            } else {
+                cmd.arg("-lc");
+            }
+            cmd.arg("-lunwind");
+            cmd.arg("-OReleaseSafe");
+            cmd.arg("-fstrip");
+            cmd.arg("-dead_strip");
+            cmd.arg("-dead_strip_dylibs");
+            cmd.arg("-fno-compiler-rt");
+            cmd.arg(&format!("-femit-bin={}", output_path.display()));
+
+            cmd.arg(&object_path);
+            cmd.arg(&c_src_path);
+            cmd.arg(libwasmer_path.join(&lib_filename));
+            if zig_triple.contains("windows") {
+                let mut libwasmer_parent = libwasmer_path.clone();
+                libwasmer_parent.pop();
+                cmd.arg(libwasmer_parent.join("winsdk/ADVAPI32.lib"));
+                cmd.arg(libwasmer_parent.join("winsdk/BCRYPT.lib"));
+                cmd.arg(libwasmer_parent.join("winsdk/KERNEL32.lib"));
+                cmd.arg(libwasmer_parent.join("winsdk/USERENV.lib"));
+                cmd.arg(libwasmer_parent.join("winsdk/WS2_32.lib"));
```

Comment:
Do we have a list of these files?

It might be a good idea to pull them into a constant at the top of the file, that way we can set up tests to make sure all the `*.lib` files are always included with Wasmer.

Reply:
No, it's just the libraries that I found out that libwasmer.a depends on.

There is no way to find out the libraries, except for using Google to lookup which DLL the missing functions belong to.

Reply:
Oh no, what I meant was instead of copy-pasting the `cmd.arg(libwasmer_parent.join(...))` line for every library, we should store them in a constant and use a loop. 

```rust
const VENDORED_LIBRARIES: &[&str] = &[
    "ADVAPI32.lib",
    "BCRYPT.lib",
    ...
];
```

That way we can add a test to the `wasmer` CLI that makes sure all the libraries are embedded in the installer. It'd be annoying if the list of libraries we tell the compiler to link against and the list of libraries bundled with `wasmer` got out of sync. We probably wouldn't notice until weeks later when our first user wants to do networking and finds we forgot to give them `WS2_32.lib`.

Reply:
I've made a loop with `fs::read_dir` now, so we don't need to hardcode any paths, it will just link to every object in the `/winsdk` folder.